front/src/common/Map/Map.scss

```
@@ -12,6 +12,17 @@
     margin-bottom: 0.5rem;
   }
 }
+.btn-map-container-editor {
+  position: absolute;
+  z-index: 2;
+  right: 25px;
+  top: 1rem;
+  button {
+    display: block;
+    font-size: 1.5rem;
+    margin-bottom: 0.5rem;
+  }
+}
```

There is a lot of repeated css here. You can adapt the previous class like that : 

```suggestion
.btn-map-container {
  position: absolute;
  z-index: 2;
  right: 25px;
  top: 5rem;
  button {
    display: block;
    font-size: 1.5rem;
    margin-bottom: 0.5rem;
  }

  &.editor {
    top: 1rem;
  }
}
```